### DISCUSSION TITLE
IP spoof

### DISCUSSION THREAD

**Discussion by 0x107u5** [leoisrael/Ghost_Dir#6] [Ideas]
adicionar uma flag possibilitando o uso de uma rede de proxy para fazer o spoof de ip tornado mais dificil sistemas de detecção e correlação, analisarem o uso da ferramenta

**Comment by leoisrael** [comment]
Adorei sua sugestão. Vou adicionar a sua ideia e tenho certeza de que isso vai fazer uma diferença incrível.
Vlw   👍
